.cirrus.yml

```diff
@@ -187,7 +199,10 @@ centos9_build_task:
         image: quay.io/libpod/centos9rust
     script:
         - cargo build
-
+podman_e2e_test:
+    alias: podman_e2e
+    depends_on: build
+    
```

Comment:
This will do nothing (does not end in `_task` and has no scripts or anything), what is it?

Reply:
left over turd